### DISCUSSION TITLE
configure librtas check

### DISCUSSION THREAD

**Discussion by mator** [util-linux/util-linux#2527] [General]
@karelzak 

why do we have empty braces for librtas (rtas) check in `configure.ac` ?

I have the following error on running `./configure` script with `librtas-dev` package installed on a debian system:
```
$ ./configure
...
checking for rtas_get_sysparm in -lrtas... yes
./configure: line 27623: ,: command not found
checking for struct sockaddr.sa_len... no
...
```
The following `configure.ac` code:
```
    717 AC_CHECK_LIB([rtas], [rtas_get_sysparm], [
    718         RTAS_LIBS="-lrtas"
    719         AC_DEFINE_UNQUOTED([HAVE_LIBRTAS], [1], [Define if librtas exists]), [],
    720 ])
    721 AC_SUBST([RTAS_LIBS])
```
notice the empty `[]` braces in the end.
I believe, we need to remove this braces:
```
util-linux$ git diff
diff --git a/configure.ac b/configure.ac
index 1c2cb4f0d..eb56c83ca 100644
--- a/configure.ac
+++ b/configure.ac
@@ -716,7 +716,7 @@ AS_IF([test x"$have_timer" = xno], [

 AC_CHECK_LIB([rtas], [rtas_get_sysparm], [
        RTAS_LIBS="-lrtas"
-       AC_DEFINE_UNQUOTED([HAVE_LIBRTAS], [1], [Define if librtas exists]), [],
+       AC_DEFINE_UNQUOTED([HAVE_LIBRTAS], [1], [Define if librtas exists])
 ])
 AC_SUBST([RTAS_LIBS])
 ```
removing them and running `./autogen.sh` and `./configure` does not produce error anymore (not depending on `librtas-dev` installed or not).

**Comment by karelzak** [comment]
Fixed by https://github.com/util-linux/util-linux/commit/b53ba174c158d5eb595b86c5fbb83d6fc433075d
